jwst/jump/twopoint_difference.py

```diff
@@ -67,7 +67,7 @@ def find_CRs(data, gdq, read_noise, rej_threshold, nframes):
         number_sat_groups = (sat_groups * 1).sum(axis=2)
         ndiffs = ngroups - 1
         sort_index = np.argsort(positive_first_diffs)
```

Comment:
Is it OK to still be using `positive_first_diffs` here, rather than `first_diffs`?